bin/phpunit

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env php
-<?php
-
-if (!\ini_get('date.timezone')) {
-    \ini_set('date.timezone', 'UTC');
-}
-
-if (\is_file(\dirname(__DIR__) . '/vendor/phpunit/phpunit/phpunit')) {
-    \define('PHPUNIT_COMPOSER_INSTALL', \dirname(__DIR__) . '/vendor/autoload.php');
-    require PHPUNIT_COMPOSER_INSTALL;
-    PHPUnit\TextUI\Command::main();
```

Comment:
This class does not exist anymore, that's why CI was failing.
There is no added value of this custom bin file over `vendor/bin/phpunit`

Reply:
will check this with the symfony recipe. because if we don't provide it the flex recipe will automatically create it.